django/core/management/color.py

```diff
@@ -13,7 +13,11 @@
 
     # Avoid initializing colorama in non-Windows platforms.
     colorama.just_fix_windows_console()
-except (ImportError, OSError):
+except (
+    AttributeError,  # colorama <= 0.4.6.
+    ImportError,  # colorama is not installed.
+    OSError,
```

Comment:
I confirmed that `OSError` still seems relevant.
https://code.djangoproject.com/ticket/32740
https://github.com/tartley/colorama/blob/7f1a5933851363abbdc24a51e76829bb23a2793b/colorama/initialise.py#L86

add comment: `# If just_fix_windows_console() accesses sys.stdout with WSGIRestrictedStdout.`